packages/storefront/src/pages/styles/border.md

```diff
@@ -4,7 +4,7 @@
 
 ## Example
 
-<Playground :frameworkMarkup="codeExample">
+<Playground :frameworkMarkup="codeExample" :externalStackBlitzDependencies="['styled-components']">
```

Comment:
Isn't this external StackBlitz dependency also shown in the Angular StackBlitz package.json example?

Reply:
No, it isn't part of the mapping.